examples/demo_mock.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "wkR0YvENQni4"
+      },
+      "source": [
+        "**Setup dependencies**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "n1sQVa5mkMEA",
+        "outputId": "dbfa09f5-0fd9-4a51-f728-fc62230f1b84"
+      },
+      "outputs": [],
+      "source": [
+        "!git clone https://github.com/AisenGinn/SMARTS 2> /dev/null\n",
+        "!cd SMARTS && ls && git checkout iss1436 && pip install .[camera-obs,colab]"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ubc6jEAqiCEq",
+        "outputId": "16e9facc-86bc-4f72-a4f2-2d8bb2961435"
+      },
+      "outputs": [],
+      "source": [
+        "import sys\n",
+        "sys.path.insert(0, \"./SMARTS/examples/env\")\n",
+        "sys.path.insert(0, \"./SMARTS/zoo\")\n",
+        "import baseline\n",
+        "import policies.keep_lane_agent"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "s7UtcphinvNv"
+      },
+      "source": [
+        "**Import Base Modules**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "id": "HsnDWYx_ngdc"
+      },
+      "outputs": [],
+      "source": [
+        "import gym\n",
+        "\n",
+        "from smarts.zoo import registry\n",
+        "from smarts.env.wrappers.episode_logger import EpisodeLogger\n",
+        "from smarts.env.wrappers.record import AgentCameraRGBRender"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LFoG7Z-FobPP"
+      },
+      "source": [
+        "**Run an episode**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 501
+        },
+        "id": "BBWj3wbAso3J",
+        "outputId": "63c1d665-b4a5-43b5-d926-b02b5bcce08f"
+      },
+      "outputs": [],
+      "source": [
+        "from smarts.core.utils.episodes import episode_range\n",
+        "from smarts.env.wrappers.record_video import RecordVideo\n",
+        "\n",
+        "#FormatObs should already be applied\n",
+        "env = gym.make(\"figure_eight-v0\")\n",
```

Comment:
It might be a little difficult for users to figure out that the `figure_eight-v0` env was registered by performing `import examples.env.baseline`. 

Here, we can do two things:

1. In notebooks, it is better to have the `imports` close to and in the same cell where they are first used. Consider moving the `import examples.env.baseline` to this cell.
2. Explicitly register the `figure_eight-v0` env again in this cell.